configure.ac

```diff
@@ -101,6 +101,7 @@ else
 -I${freebsd_src}/cddl/compat/opensolaris/lib/libumem
 -I${freebsd_src}/cddl/contrib/opensolaris/lib/libzpool/common
 -I${freebsd_src}/sys/cddl/contrib/opensolaris/common/zfs
+-I${freebsd_src}/sys/cddl/contrib/opensolaris/uts/common/fs/zfs/sys
```

Comment:
Is this still necessary after the `zfs_ioctl.h` fix in master ? I think not ? Can you please confirm ?

Reply:
You're right.  I'll remove it.